### DISCUSSION TITLE
SA, why not include density in material derivative?

### DISCUSSION THREAD

**Discussion by bigfooted** [su2code/SU2#1790] [General]
At the nasa website, 
https://turbmodels.larc.nasa.gov/spalart.html

they mention that you can write SA in the form 

$\frac{\partial \rho \nu}{\partial t} + \frac{\partial \rho u_j \nu}{\partial x_j} = ...$

So why do we not use this formulation instead of the formulation without density?

**Comment by pcarruscag** [comment]
It just looks like more work TBH, for example we would need the gradient of density for that version.
